aeon/io/reader.py

```diff
@@ -286,23 +286,35 @@ def __init__(self, pattern: str, model_root: str = "/ceph/aeon/aeon/data/process
         # `pattern` for this reader should typically be '<hpcnode>_<jobid>*'
         super().__init__(pattern, columns=None)
         self._model_root = model_root
+        self.config_file = None  # requires reading the data file to be set
```

Comment:
@glopesdev you are correct, making `config_file` a class property is misleading because the value of `config_file` is only valid within the `read` call and not reliably valid for the entire class.

The ingestion does need this but there are other workarounds - I'll make updates to fix this